drivers/platform/aducm3029/aducm3029_gpio_irq.c

```diff
@@ -542,6 +542,18 @@ static int aducm_gpio_irq_set_priority(struct no_os_irq_ctrl_desc *desc,
 	return 0;
 }
 
+/**
+ * @brief Unused
+ * @param desc - GPIO interrupt controller descriptor.
+ * @return -ENOSYS
+ */
+static int32_t aducm_gpio_irq_clear_pending(struct no_os_irq_ctrl_desc *desc,
+		uint32_t irq_id)
+{
+	return -ENOSYS;
+}
```

Comment:
yeah my bad, sorry for that, it was just an afterthought after review and i wrote it so that we keep that in mind, not to implement it in this review, as @rbolboac is saying it turns out those dummy implementations are not needed